interfaces/interface_settings.py

```diff
@@ -0,0 +1,104 @@
+# coding:utf-8
```

Comment:
File `interface_*.py` **SHOULD** be generated from `interface_*.ui`.

**SHOULD**   This word, or the adjective "RECOMMENDED", mean that there may exist valid reasons in particular circumstances to ignore a particular item, but the full implications must be understood and carefully weighed before choosing a different course.